src/components/Common/SearchByMultipleFields.tsx

```diff
@@ -30,7 +30,6 @@ import PhoneNumberFormField from "@/components/Form/FormFields/PhoneNumberFormFi
 
 interface SearchOption {
   key: string;
-  label: string;
```

Comment:
I am pretty sure that a label is required for the search option, why would we remove it 🤔

Reply:
removed its only usage